setup.cfg

```
@@ -93,6 +93,7 @@ tests =
     flake8-debugger>=4.0.0
     flake8-mutable>=1.2.0
     flake8-simplify>=0.14.0
+    flake8-type-checking; python_version > "3.7"
```

(The uiserver is Python 3.8-3.9 only)
```suggestion
    flake8-type-checking
```